frontend/tests/features/collection/collection.test.ts

```diff
@@ -91,6 +97,8 @@ describe("Collection", () => {
 async function createCollection(
   collection?: Partial<Collection>
 ): Promise<string> {
+  await goToPage(`${TEST_URL}${ROUTES.MY_COLLECTIONS}`);
```

Comment:
Datasets index page does not display "Create Collection" button; must navigate to My Collections.